### DISCUSSION TITLE
Media filter

### DISCUSSION THREAD

**Discussion by ghost** [martinrotter/rssguard#1194] [Q&A]
Greetings! I am trying to make a filter for messages containing media, but I don't see how, could someone guide me?

**Comment by martinrotter** [comment]
Give link to feed/source you want to filter and write exactly what the filter should do.

**Reply by ghost** [reply]
Thanks for replying. I would like to ignore content that does not contain images, videos, gifs, etc. This would be any nitter user url, like this one: https://nitter.privacydev.net/NASA/rss

I know nitter allows me to do it through /user/media/rss (in fact that's what I have now), but that requires more tokens and I'd like to reduce the burden for public instances.

**Reply by martinrotter** [reply]
Provided feed URL has at this point only articles which do contain images. So hard to test any solution RN. Provide some feed which has mixed articles (some with images, some without).

**Reply by ghost** [reply]
Sure! These look pretty mixed between text, images and videos:
https://nitter.privacydev.net/CNN/rss
https://nitter.nohost.network/SpaceX/rss
https://nitter.catsarch.com/imVkohli/rss

These are only videos:
https://nitter.cz/_bestvideos/rss
https://nitter.poast.org/gifs/rss

Images only:
https://nitter.ktachibana.party/toplandscapes/rss

Text only:
https://nitter.woodland.cafe/ShowerThoughtsy/rss
